lib/registry.js

```diff
@@ -82,7 +82,21 @@ class Registry {
 	}
 
 	getMetricsAsJSON() {
-		return this.getMetricsAsArray().map(metric => metric.get());
+		return this.getMetricsAsArray().map(metric => {
+			const item = metric.get();
+			if (!item.values) {
+				return item;
+			}
+
+			item.values = item.values.map(val =>
```

Comment:
this still mutates `item.values`. Should probably do a `JSON.parse(JSON.stringify(item))` or something

Reply:
metric.get returns new object always, array.map returns new array. So there is no mutation here.

If you insist i'll change this :)

Reply:
Ah, `get` returns a new object. Then this is fine, thanks!